projects/vendor/src/yotpo/yotporeview/yotporeview.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, AfterViewInit, ElementRef } from '@angular/core';
+import { BaseyotpoService } from './../baseyotpo/baseyotpo.service';
+import { Observable } from 'rxjs';
+import { Product } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-yotporeview',
+  templateUrl: './yotporeview.component.html',
```

Comment:
Typically for components and files the naming would be something more along the lines of `yotpo-review.component...` and the selector would be `cx-yotpo-review`.

Reply:
done. thanks